x-pack/plugin/core/template-resources/src/main/resources/logs@settings-logsdb.json

```diff
@@ -0,0 +1,26 @@
+{
+  "template": {
+    "settings": {
+      "index": {
```

Comment:
Do we really need an additional template? Why not use a template variable like we do for version for the index mode? 

Having more templates means more templates to keep in sync / more confusion.